### PR TITLE
 Reporting some grammatical mistakes

### PR DESCRIPTION
**Pull request by gautamjajoo**
Here are some grammatical changes in the file.
Thanks!

### CODE REVIEW

File: docs/source/architecture_decisions.md [hunk 1]
```diff
@@ -4,23 +4,23 @@ This is a collection of records for architecturally significant decisions.
 
 ### URL Patterns
 
-We follow a very basic yet strong convention for URL so that our rest APIs are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
+We follow a very basic ,yet strong convention for URL so that our rest API's are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
```
Comment (RishabhJain2018) [inline docs/source/architecture_decisions.md]: Please remove the space between `basic` & `,`. Add it between `,` and `yet` like `basic, yet`

Comment (RishabhJain2018) [inline docs/source/architecture_decisions.md]: Please the apostrophe in `API's`. It should be `APIs`

File: docs/source/architecture_decisions.md [hunk 2]
```diff
@@ -4,23 +4,23 @@ This is a collection of records for architecturally significant decisions.
...
+* `PUT /hosts/challenge_host_team/<challenge_host_team_id>` - Updates a specific challenge host team,
 
-* `PATCH /hosts/challenge_host_team/<challenge_host_team_id>` - Partially updates a specific challenge host team
+* `PATCH /hosts/challenge_host_team/<challenge_host_team_id>` - Partially updates a specific challenge host team,
 
-* `DELETE /hosts/challenge_host_team/<challenge_host_team_id>` - Deletes a specific challenge host team
+* `DELETE /hosts/challenge_host_team/<challenge_host_team_id>` - Deletes a specific challenge host team.
 
-Also, we have namespaced the URL patterns on an app basis, so URLs for _Challenge Host Model_ which is in _hosts_ app will be
+Also, we have namespaced the URL patterns on an appropriate basis, so URLs for _Challenge Host Model_ which is in _hosts_ app will be
```
Comment (RishabhJain2018) [inline docs/source/architecture_decisions.md]: Here, `app` refers to the django apps that we are using in the project. So, please revert the change.

File: docs/source/architecture_decisions.md [hunk 3]
```diff
@@ -4,23 +4,23 @@ This is a collection of records for architecturally significant decisions.
 
 ### URL Patterns
 
-We follow a very basic yet strong convention for URL so that our rest APIs are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
+We follow a very basic ,yet strong convention for URL so that our rest API's are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
 
 For example, to perform **CRUD** operation on _Challenge Host Model_, following will be the URL patterns.
 
-* `GET /hosts/challenge_host_team` - Retrieves a list of challenge host teams
+* `GET /hosts/challenge_host_team` - Retrieves a list of challenge host teams,
```
Comment (RishabhJain2018) [inline docs/source/architecture_decisions.md]: Can you please remove the comma as I think earlier version(without comma) was looking good . :-)

**Comment by gautamjajoo** [comment]
Done the minor changes also.
